packages/widget/src/hooks/useCreateEvmWallets.tsx

```diff
@@ -20,7 +20,7 @@ export const useCreateEvmWallets = () => {
   const { data: chains } = useAtomValue(skipChainsAtom);
   const { data: assets } = useAtomValue(skipAssetsAtom);
   const [sourceAsset, setSourceAsset] = useAtom(sourceAssetAtom);
-  const [evmWallet, setEvmWallet] = useSetAtom(evmWalletAtom);
+  const [evmWallet, setEvmWallet] = useAtom(evmWalletAtom);
```

Comment:
i forgot to update useSetAtom to useAtom in my previous PR https://github.com/skip-mev/skip-go/pull/836